docs/root/configuration/http/http_filters/ip_tagging_filter.rst

```diff
@@ -3,9 +3,9 @@
 IP Tagging
 ==========
 
-The HTTP IP Tagging filter sets the header *x-envoy-ip-tags* with the string tags for the trusted address from
-:ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`. If there are no tags for an address,
-the header is not set.
+The HTTP IP Tagging filter sets the header *x-envoy-ip-tags* if the alternate ``ip_tag_header`` is not provided
```

Comment:
Ideally ip_tag_header link to your new Porto field if possible

Reply:
Is that even possible given the field is being added in this PR? Let me look around

Reply:
Looks like what I tried didn't work
```
Error: ERROR: /source/docs/BUILD:273:8: Executing genrule //docs:html failed: (Exit 1): bash failed: error executing command (from target //docs:html)
Error: ERROR: Build failed. Not running target
Warning: /tmp/tmp66s3dfsy/generated/rst/version_history/v1.33/v1.33.0.rst:45: WARNING: undefined label: 'envoy_v3_api_msg_extensions.filters.http.ip_tagging.v3.iptagging.ip_tag_header'
```

any pointers on how to do this will be very helpful 🙇🏽